docs/api/responses/meta.md

```diff
@@ -2,72 +2,72 @@
 
-``type`` - **required** - type of the content; e.g. `movie`, `series`, `channel`, `tv` (see [Content Types](./content.types.md))
+``type`` - **required** - string, type of the content; e.g. `movie`, `series`, `channel`, `tv` (see [Content Types](./content.types.md))
 
-``name`` - **required** - name of the content
+``name`` - **required** - string, name of the content
 
-``genres`` - _optional_  - genre/categories of the content; array of strings, e.g. ``["Thriller", "Horror"]``
+``genres`` - _optional_  - array of strings, genre/categories of the content; e.g. ``["Thriller", "Horror"]``
 
-``poster`` - _optional_ - URL to png of poster; accepted aspect ratios: 1:0.675 (IMDb poster type) or 1:1 (square) ; you can use any resolution, as long as the file size is below 100kb; below 50kb is recommended
+``poster`` - _optional_ - string, URL to png of poster; accepted aspect ratios: 1:0.675 (IMDb poster type) or 1:1 (square) ; you can use any resolution, as long as the file size is below 100kb; below 50kb is recommended
 
-``posterShape`` - _optional_ - can be `square` (1:1 aspect) or `regular` (1:0.675) or `landscape` (1:1.77). If you don't pass this, `regular` is assumed
+``posterShape`` - _optional_ - string, can be `square` (1:1 aspect) or `regular` (1:0.675) or `landscape` (1:1.77). If you don't pass this, `regular` is assumed
 
-``background`` - _optional_ - the background shown on the stremio detail page ; heavily encouraged if you want your content to look good; URL to PNG, max file size 500kb
+``background`` - _optional_ - string, the background shown on the stremio detail page ; heavily encouraged if you want your content to look good; URL to PNG, max file size 500kb
 
-``logo`` - _optional_ - the logo shown on the stremio detail page ; encouraged if you want your content to look good; URL to PNG
+``logo`` - _optional_ - string, the logo shown on the stremio detail page ; encouraged if you want your content to look good; URL to PNG
 
-``description`` - _optional_ - a few sentances describing your content
+``description`` - _optional_ - string, a few sentances describing your content
 
-``releaseInfo`` - _optional_ - string - year the content came out ; if it's ``series`` or ``channel``, use a start and end years split by a tide - e.g. ``"2000-2014"``. If it's still running, use a format like ``"2000-"``
+``releaseInfo`` - _optional_ - string, year the content came out ; if it's ``series`` or ``channel``, use a start and end years split by a tide - e.g. ``"2000-2014"``. If it's still running, use a format like ``"2000-"``
 
-``director``, ``cast`` - _optional_  - directors and cast, both arrays of names
+``director``, ``cast`` - _optional_  - directors and cast, both arrays of names (string)
 
-``imdbRating`` -  _optional_ - IMDb rating, a number from 0 to 10 ; use if applicable
+``imdbRating`` -  _optional_ - string, IMDb rating, a number from 0.0 to 10.0 ; use if applicable
```

Comment:
isn't this a number?

Reply:
i checked a meta response from cinemeta before writing "string", cinemeta returns them as strings